packages/peregrine/src/List/__tests__/items.spec.js

```diff
@@ -6,11 +6,38 @@ import { Items } from '..';
 
 configure({ adapter: new Adapter() });
 
-const items = Object.entries({
-    a: { id: 'a', val: '10' },
-    b: { id: 'b', val: '20' },
-    c: { id: 'c', val: '30' }
-});
+const items = [
+    [
+        1,
+        {
+            id: 1,
```

Comment:
The tests for `List` and its child components have been updated as of #62, using almost the same sample data you've got here. You can probably delete these test changes.

Also, I think it's better to have `id` be a `string` instead of a `number`. Is that something we can accomplish with our query?

Reply:
You can't cast in a GraphQL query, so we'd have to cast on the client anytime that data came in. That would be a PITA, because any query that takes `id` as an arg would have to be cast back to a number.